tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -807,7 +807,7 @@ public:
    /// ~~~
    ///
    /// This action is *lazy*: upon invocation of this method the calculation is
-   /// booked but not executed. See RResultPtr documentation.
+   /// booked but not executed. Also see RResultPtr.
```

Comment:
```suggestion
   /// booked but not executed. Also see ROOT::RDF::RResultPtr.
```

You want the full specifier with namespaces such as the change before, or just the class name?

Reply:
Same for following changes. But I guess it's the doxygen `\return` markup, which should be more technical?

Reply:
In `RInterface.hxx` I'm using the unqualified `RResultPtr` everywhere (or at least that's my intention) because `RInterface` is also in the `ROOT::RDF` namespace so the unqualified name is enough for Doxygen to make the type name a link.

Elsewhere I need the fully qualified name otherwise Doxygen does not know what type I'm talking about.